### PR TITLE
fix Installing Kubeflow link

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/kubeflow/website/issues/2907

### CODE REVIEW

File: content/en/docs/distributions/operator/install-kubeflow.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = "Instructions for Kubeflow deployment with Kubeflow Operator"
 weight = 10
 +++
 
-This guide describes how to use the Kubeflow Operator to deploy Kubeflow. As mentioned in the Operator [introduction](/docs/methods/operator/introduction.md), the Operator also allows you to monitor and manage the Kubeflow installation beyond the initial installation.
+This guide describes how to use the Kubeflow Operator to deploy Kubeflow. As mentioned in the Operator [introduction](/docs/distributions/operator/introduction.md/), the Operator also allows you to monitor and manage the Kubeflow installation beyond the initial installation.
```
Comment (shawnzhu) [inline content/en/docs/distributions/operator/install-kubeflow.md:7]: it works for me if removing the trailing slash

Comment (Arhell) [reply content/en/docs/distributions/operator/install-kubeflow.md:7]: Thanks, I will correct

**Comment by Bobgy** [comment]
Is this ready for review? Looks like a draft

**Comment by Bobgy** [comment]
https://github.com/kubeflow/website/blob/master/content/en/docs/distributions/operator/OWNERS
Hi @Arhell, you should ask the kubeflow operator's owner to review the change.

/assign @animeshsingh

**Pull request closed without merging**